### PR TITLE
Fixes #37128 - Unpin @adobe/css-tools

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
Leaving it in `package-exclude.json` as some plugins still pin it.

File: package.json [hunk 1]
```diff
@@ -29,7 +29,6 @@
     "react-intl": "^2.8.0"
   },
   "devDependencies": {
-    "@adobe/css-tools": "~4.2.0",
```
Comment (evgeni) [inline package.json:32]: this fixes CVE-2023-26364 and CVE-2023-48631

**Review by ekohl — COMMENTED** [review]
The commit message says Unpin, but isn't this just dropping the whole dependency? Is it actually unused now or somehow pulled in via `@theforeman` packages?

**Comment by evgeni** [comment]
> The commit message says Unpin, but isn't this just dropping the whole dependency? Is it actually unused now or somehow pulled in via `@theforeman` packages?

It's not a direct depenendency, yeah. Something (I didn't check what exactly, it's not in the bundled packaged modules either) pulls it in and we pinned it for Node12 compat.

**Review by ekohl — APPROVED** [review]

**Pull request merged**